controllers/nginx/Changelog.md

```diff
@@ -2,6 +2,95 @@ Changelog
 
 ### 0.9
 
+**Temporal Image:**  `quay.io/aledbf/nginx-ingress-controller:0.9.0`
+
+*New Features:*
+
+- SSL Passthrough
+- New Flag `--publish-service` that set the Service fronting the ingress controllers
+- Ingress status shows the correct IP/hostname address without duplicates
+- Custom body sizes per Ingress
+- Prometheus metrics
+
+
+*Breaking changes:*
+
+- Flag `--nginx-configmap` was replaced with `--configmap`
+- Configmap field `body-size` was replaced with `proxy-body-size`
```

Comment:
Please update the releases page per https://github.com/kubernetes/ingress/blob/master/docs/dev/releases.md#releasing, with the new features and breaking changes, and link to this page for people who want a more complete list of prs/changes.